src/jdk.crypto.ec/share/native/libsunec/impl/mpi.c

```diff
@@ -1727,17 +1727,28 @@ int    mp_cmp_mag(mp_int *a, mp_int *b)
  */
 int    mp_cmp_int(const mp_int *a, long z, int kmflag)
 {
-  mp_int  tmp;
-  int     out;
-
+  mp_sign signz = z < 0 ? ZNEG : ZPOS;
+  
```

Comment:
Please remove the trailing whitespace here.
This file should have an IBM copyright notice.

Reply:
Updated and running new testing via https://openj9-jenkins.osuosl.org/job/Pipeline-Build-Test-Personal/451/

Reply:
```
[2023-11-08T22:05:46.059Z] /home/jenkins/workspace/Build_JDK11_s390x_linux_Personal/src/jdk.crypto.ec/share/native/libsunec/impl/mpi.c: In function 'mp_cmp_int':
[2023-11-08T22:05:46.059Z] /home/jenkins/workspace/Build_JDK11_s390x_linux_Personal/src/jdk.crypto.ec/share/native/libsunec/impl/mpi.c:1736:27: error: 'ZNEG' undeclared (first use in this function); did you mean 'NEG'?
```
`ZNEG` should be `NEG` (see e.g. line 376); I suppose `NEG` means "negative" and `ZPOS` means "zero or positive".

Reply:
https://openj9-jenkins.osuosl.org/job/Pipeline-Build-Test-Personal/452/